### PR TITLE
Style elements

### PR DESCRIPTION
**Pull request by HelenHMaher**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,12 +1,12 @@
-@import url("https://rsms.me/inter/inter.css");
```
Comment (HelenHMaher) [inline src/App.css:1]: the transparency hadn't been imported correctly so I had to change that to make them work properly.

File: src/App.js [hunk 2]
```diff
@@ -3,55 +3,54 @@ import './App.css';
 import Dashboard from './components/Dashboard/Dashboard';
```
Comment (HelenHMaher) [inline src/App.js:3]: these are all just formatting changes

File: src/Redux/userSlice.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ export const fetchUser = createAsyncThunk('user/fetchUser', async () => {
     });
```
Comment (HelenHMaher) [inline src/Redux/userSlice.js:15]: getting rid of an error message (if no user is detected "no user" will be stored in the redux store)

File: src/components/BottomNav/RippleButton.css [hunk 4]
```diff
@@ -23,7 +23,7 @@
   border-radius: 100%;
```
Comment (HelenHMaher) [inline src/components/BottomNav/RippleButton.css:23]: chasing down the last of the old color variables

File: src/components/Dashboard/Dashboard.js [hunk 5]
```diff
@@ -1,34 +1,33 @@
 import React, { useEffect } from 'react';
```
Comment (HelenHMaher) [inline src/components/Dashboard/Dashboard.js:1]: cleaning up some of the warnings that are being logged to the console

File: src/components/Profile/index.js [hunk 6]
```diff
@@ -1 +1 @@
-// export Profile from './Profile';
```
Comment (HelenHMaher) [inline src/components/Profile/index.js:1]: cleaning up those pesky error messages

File: src/components/Start/Start.js [hunk 7]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import './Start.css';
```
Comment (HelenHMaher) [inline src/components/Start/Start.js:2]: coding the Start button

File: src/components/TestLogin/TestLogin.js [hunk 8]
```diff
@@ -4,116 +4,124 @@ import { useHistory } from 'react-router-dom';
 // import "./TestLogin.css";
 
 export default function TestLogin() {
-	const [ username, setUsername ] = useState('');
```
Comment (HelenHMaher) [inline src/components/TestLogin/TestLogin.js:7]: Ugh, just ignore this TestLogin shit.  It's fine -_-.

**Review by FilipNew1234 — APPROVED** [review]
I see a lot of 'Preetify changes' 
Hopefully it doesnt affect the code....

**Pull request merged**